### DISCUSSION TITLE
how-did-I-become-a-front-end-developer-from-a-java-developer/

### DISCUSSION THREAD

**Discussion by giscus** [zxuqian/zxuqian.cn#81] [Announcements]
# how-did-I-become-a-front-end-developer-from-a-java-developer/

前端作为比较新的技术岗位，涉及范围广泛，市面上也没有成体系的、适用于所有人的学习方法。这篇文章，将分享一下我转型成为前端工程师的经历，以及我在学习前端开发过程中，遇到的问题和总结出来的一些经验，希望对你有所启发。

https://zxuqian.cn/how-did-I-become-a-front-end-developer-from-a-java-developer/

**Comment by jiayit** [comment]
很棒！
